### PR TITLE
add sample NewsArticle

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@juliantoledo @kul3r4 PTAL

File: src/60_Sample_AMPs/News_Article.html [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    Mutat dolores ea duo. Usu cu dico everti tincidunt, pri vero nullam vidisse no. Eam novum aeque populo cu. Verear contentiones ne quo, no sed consulatu definitionem.
+    </p>
+
+    <!--
+    Use `amp-carousel` for image galleries. Learn more about creating image gallieries in AMP [here](/advanced/image_galleries_with_amp-carousel/).
+    -->
+    <amp-carousel width="1280" height="970" layout="responsive" type="slides"> 
+      <figure>
+        <amp-img src="/img/landscape2.jpg" width=1280 height=853 layout=responsive></amp-img>
+        <figcaption>Here is an image gallerie..</figcaption>
```
Comment (kul3r4) [inline src/60_Sample_AMPs/News_Article.html]: Typo: gallerie  -> gallery

File: src/json/related_articles.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ "items": [
+   {
+     "title": "Shopping websites with AMP",
+     "thumbnail": "/img/shopping_thumb.jpg",
+     "url": "https://ampbyexample.com/samples/ecommerce"
+   },
+   {
+     "title": "The tastiest recipes with AMP",
+     "thumbnail": "/img/recipe_thumb.jpg",
+     "url": "https://ampbyexample.com/samples/recipes"
```
Comment (kul3r4) [inline src/json/related_articles.json]: Is this new page in this pull request?

File: src/60_Sample_AMPs/News_Article.html (hunk 1 above)
Comment (sebastianbenz) [inline src/60_Sample_AMPs/News_Article.html]: done

File: src/json/related_articles.json (hunk 2 above)
Comment (sebastianbenz) [inline src/json/related_articles.json]: nope - Boris is working the product samples. I removed the other two links.

**Comment by sebastianbenz** [comment]
PTAL

File: src/60_Sample_AMPs/News_Article.html [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+ }
+      </style>
+
+  </head>
+  <body>
+
+    <!-- -->
+    <div class="logo">
+      <a href="/" id="name"> AMP by Example</a>
+      <div id="sections">Introduction | Components | Advanced | Samples</div>
```
Comment (kul3r4) [inline src/60_Sample_AMPs/News_Article.html]: How we are using this sections?

File: src/60_Sample_AMPs/News_Article.html [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+    </div>
+
+    <div class="heading">
+      <p>Samples &gt; News Article</p>
+      <h1>A sample AMP Article</h1>
+      <p id="summary">This is a sample article demonstrating how to write a news article in AMP. It demonstrates the usage of AMP components which works well in news articles. Examples include social sharing, image galleries, personalized content, ads, and more.</p>
+      <p><small>by Sebastian Benz</small></p>
+    </div>
+
+    <!-- #### Social sharing...  -->
```
Comment (kul3r4) [inline src/60_Sample_AMPs/News_Article.html]: Why "..."?

**Comment by kul3r4** [comment]
Just 2 comments, otherwise LGTM.

File: src/60_Sample_AMPs/News_Article.html (hunk 3 above)
Comment (sebastianbenz) [inline src/60_Sample_AMPs/News_Article.html]: Not at all. Was meant as a stylistic element, removed it.

File: src/60_Sample_AMPs/News_Article.html (hunk 4 above)
Comment (sebastianbenz) [inline src/60_Sample_AMPs/News_Article.html]: done

**Comment by sebastianbenz** [comment]
Fixed two issues. Also restyled and simplified the header. 

PTAL

**Comment by sebastianbenz** [comment]
Sample: https://amp-by-example-staging.appspot.com/sample_amps/news_article/preview/

**Comment by kul3r4** [comment]
LGTM.

**Comment by sebastianbenz** [comment]
Thanks for review Chiara!

**Pull request merged**